contents.json

```diff
@@ -21968,6 +21968,26 @@
       "stars": 93,
       "date_added": "Jul 5 2020",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "Sketch Elements",
+      "description": "SwiftUI starter kit based on Sketch Elements",
+      "category-ids": [
+        "swiftui"
+      ],
+      "source": "https://github.com/molcik/ios-sketch-elements",
+      "license": "other",
+      "tags": [
+        "swift",
+        "swiftui"
+      ],
+      "stars": 0,
+      "date_added": "Jul 6 2020",
+      "suggested_by": "@molcik",
+            "screenshots": [
+        "https://filipmolcik.com/wp-content/uploads/2020/07/iphonexspacegrey_portrait-2-1024x1024.png",
+        "https://filipmolcik.com/wp-content/uploads/2020/07/iphonexspacegrey_landscape-2-1024x1024.png"
+      ],
```

Comment:
There should not be a comma here, could you remove it?

Reply:
Sure